gui/main_process/encrypt.js

```diff
@@ -56,4 +61,18 @@ ipcMain.on('encrypt', (event, input_path, output_path) => {
       event.reply('notify-encrypt-error', result);
     }
   });
+});
+
+ipcMain.on('encrypt-cancel', (_event) => {
+  try {
+    const pid = fs.readFileSync(path.join(basepath, 'pid'), 'utf8');
+    process.kill(pid);
+    var currentWindow = BrowserWindow.getFocusedWindow();
+    currentWindow.close();
+    client.invoke("cleanup", function(_error, _res) {
+    
+    });
+  } catch (err) {
```

Comment:
Is there a good way to let the user know that stopping encryption failed for some reason?